HISTORY.md

```diff
@@ -163,6 +163,10 @@ Standard library changes
   results table ([#38042]).
 * `@testset` now supports the option `verbose` to show the test result summary
   of the children even if they all pass ([#33755]).
+* In `LinearIndices(::Tuple)` and `CartesianIndices(::Tuple)`, integer elements (as opposed to ranges of integers) of the
+  argument tuple now always describe 1-based ranges, e.g, `CartesianIndices((3, 1:3))` is equivalent to
+  `CartesianIndices((1:3, 1:3))`. This is consistent with the (old and new) behavior for tuples containing only integers
+  and is considered a bugfix ([#37829], [#37928]).
```

Comment:
How about this?  It's a little tricky to _both_ keep with the style of the NEWS (leading with the new behavior) and also lead with the bugfix (which — by definition — leads with the old behavior).  Here's a shot at walking down the middle:

```suggestion
* In `LinearIndices(::Tuple)` and `CartesianIndices(::Tuple)`, integers (as opposed to ranges of integers) in the
  argument tuple now consistently describe 1-based ranges, e.g, `CartesianIndices((3, 1:3))` is equivalent to
  `CartesianIndices((1:3, 1:3))`. This is how tuples of integers have always been documented to work, but a
  bug had caused erroneous behaviors with heterogeneous tuples containing both integers and ranges ([#37829], [#37928]).
```

Reply:
I like it.